docs/controls/access_control/ac-02.md

```diff
@@ -0,0 +1,60 @@
+# Access Control
+## AC-02 - Account Management
+The organization:
+
+a. Identifies and selects the following types of information system accounts to support organizational missions/business functions: (individual, shared, group, system, guest/anonymous, emergency, developer/manufacturer/vendor, temporary, and service);
+
+b. Assigns account managers for information system accounts;
+
+c. Establishes conditions for group and role membership;
+
+d. Specifies authorized users of the information system, group and role membership, and access authorizations (i.e., privileges) and other attributes (as required) for each account;
+
+e. Requires approvals by (applicable information system managers) for requests to create information system accounts;
+
+f. Creates, enables, modifies, disables, and removes information system accounts in accordance with (the processes defined in the OS Standard Operating Procedures for Information Security: Access Control document (see Section AC-2));
+
+g. Monitors the use of information system accounts;
+
+h. Notifies account managers:
+
+   1. When accounts are no longer required;
+   2. When users are terminated or transferred; and
+   3. When individual information system usage or need-to-know changes;
+
+i. Authorizes access to the information system based on:
+
+   1. A valid access authorization;
+   2. Intended system usage; and
+   3. Other attributes as required by the organization or associated missions/business functions;
+
+j. Reviews accounts for compliance with account management requirements [at least every 180 days]; and
+
+k. Establishes a process for reissuing shared/group account credentials (if deployed) when individuals are removed from the group.
+
+For CSP Only: AC-2 (j) [at least annually]
+
+## TDP Implementation
+a. The TDP system accounts are OFA Admin and Data Prepper.  
+
+b. The OFA Admin is responsible for managing the system accounts.  The OFA Admin approves new users, updates profile information, deactivates, and reactivates users.
+
+c. Members in the roles of OFA Admin and Data Preppers are based on their job responsibilities.  The activities of each of the roles can be found in [AC-05](https://github.com/raft-tech/TANF-app/blob/documentation/access-controls-ac_5/docs/controls/access_control/ac-05.md)
```

Comment:
Based on this [comment](https://github.com/HHS/TANF-app/pull/174#discussion_r573264892) there is also another group of users that exists higher in the hierarchy that includes `System/Product Owner` and `Tech Lead`.

Reply:
These have been updated in AC-5.  Also, added System Admin.

Reply:
@hursey013 The update here looks good to me - let me know if you need anything else